openpype/hosts/traypublisher/plugins/publish/collect_clip_instances.py

```diff
@@ -0,0 +1,38 @@
+from pprint import pformat
+import pyblish.api
+
+
+class CollectClipInstance(pyblish.api.InstancePlugin):
+    """Collect clip instances and resolve its parent"""
+
+    label = "Collect Clip Instances"
+    order = pyblish.api.CollectorOrder - 0.081
+
+    hosts = ["traypublisher"]
+    families = ["plate", "review", "audio"]
+
+    def process(self, instance):
+        creator_identifier = instance.data["creator_identifier"]
+        if (
+            creator_identifier not in [
+                "editorial_plate",
+                "editorial_audio",
+                "editorial_review"
+            ]
+        ):
```

Comment:
This can be reduced
```suggestion
        if creator_identifier not in [
            "editorial_plate",
            "editorial_audio",
            "editorial_review"
        ]:
```